BUILD.gn

```diff
@@ -22,27 +22,53 @@ shared_library("zoslib") {
   output_name = "zoslib"
 
   sources = [
-    "include/csrsic.h",
-    "include/edcwccwi.h",
-    "include/zos.h",
-    "include/zos-base.h",
-    "include/zos-bpx.h",
-    "include/zos-char-util.h",
-    "include/zos-getentropy.h",
-    "include/zos-io.h",
-    "include/zos-savstack.h",
-    "include/zos-semaphore.h",
-    "include/zos-setlibpath.h",
-    "include/zos-sys-info.h",
-    "include/zos-tls.h",
-    "src/zos.cc",
     "src/zos-bpx.cc",
     "src/zos-char-util.cc",
     "src/zos-getentropy.cc",
     "src/zos-io.cc",
     "src/zos-semaphore.cc",
+    "src/zos-spawn.cc",
     "src/zos-sys-info.cc",
     "src/zos-tls.cc",
+    "src/zos.cc",
+    "src/zoslib-help.cc",
+    "include/csrsic.h",
```

Comment:
for future enhancement, we should put the include files in a separate list and then have a targets for generating the header files & installing them.

Reply:
Not sure I understand.. I used the same approach as done in V8's BUILD.gn. As shown in the [GN](https://gn.googlesource.com/gn/+/main/docs/quick_start.md) examples, the only attribute for source files, including headers, is "sources".

For installation, we have:
```
install(
    DIRECTORY ${CMAKE_SOURCE_DIR}/include/
    DESTINATION "include"
    FILES_MATCHING PATTERN "*")
```
which works fine including for subdirs.